parts/k8s/kubernetesinstalls.sh

```diff
@@ -99,36 +98,45 @@ function installClearContainersRuntime() {
 }
 
 function installNetworkPlugin() {
-    ls $CNI_BIN_DIR
-    if [ $? -eq 0 ]; then
-        echo "network plugin already installed, skipping download"
-    else
-        if [[ "${NETWORK_PLUGIN}" = "azure" ]]; then
-            installAzureCNI
-        elif [[ "${NETWORK_PLUGIN}" = "kubenet" ]]; then
-            installCNI
-        elif [[ "${NETWORK_PLUGIN}" = "flannel" ]]; then
-            installCNI
-        fi
+    if [[ "${NETWORK_PLUGIN}" = "azure" ]]; then
+        installAzureCNI
     fi
+    installCNI
+}
+
+function downloadCNI() {
+    mkdir -p $CNI_DOWNLOADS_DIR
+    CNI_TGZ_TMP=$(echo ${CNI_PLUGINS_URL} | cut -d "/" -f 5)
```

Comment:
lol

Reply:
is that bad? 😬

Reply:
Laughter is the best medicine, so very very good.